### PR TITLE
[Breadcrumbs update] HeaderLayout subtitle node support

### PR DESCRIPTION
**Pull request by ronronscelestes**
## What

- Extending HeaderLayout subtitle type support needed for ML folders

## Snapshot

image

### CODE REVIEW

File: packages/strapi-design-system/src/Layout/HeaderLayout.js [hunk 1]
```diff
@@ -81,9 +83,13 @@ export const BaseHeaderLayout = React.forwardRef(
                 <Typography variant="beta" as="h1" {...props}>
                   {title}
```
Comment (ronronscelestes) [inline packages/strapi-design-system/src/Layout/HeaderLayout.js:84]: I was wondering about allowing the title to be a node too. My only worry is that so far we make sure we have an `h1` on the page, which wouldn't be the case allowing a node, wdyt @gu-stav?

Comment (gu-stav) [reply packages/strapi-design-system/src/Layout/HeaderLayout.js:84]: I'd say the design-system should be as strict as possible. If we don't need to open up that slot for nodes, I'd keep it as is.

File: packages/strapi-design-system/src/Layout/HeaderLayout.js [hunk 2]
```diff
@@ -137,7 +147,7 @@ BaseHeaderLayout.propTypes = {
   primaryAction: PropTypes.node,
   secondaryAction: PropTypes.node,
   sticky: PropTypes.bool,
-  subtitle: PropTypes.string,
+  subtitle: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```
Comment (gu-stav) [inline packages/strapi-design-system/src/Layout/HeaderLayout.js:150]: Could you add a comment:

```
// TODO V2: Remove the string fallback
```

to make sure we remove that in the next major iteration?

**Review by gu-stav — APPROVED** [review]
Great, thanks!

**Pull request merged**